drivers/hmis/app/graphql/types/hmis_schema/query_type.rb

```diff
@@ -209,5 +209,21 @@ def denied_pending_referral_postings(**args)
 
       scoped_referral_postings(postings, **args)
     end
+
+    # AC HMIS Queries
+
+    field :esg_funding_report, Types::AcHmis::EsgFundingReport, null: false do
```

Comment:
Why add an intermediate type rather than resolving  an array like `[Types::AcHmis::EsgFundingService]`? This seems fine too, just wondering.

Reply:
@gigxz The PT issue specified an intermediate type:

```graphql
esgFundingReport(clientIds: [1, 2, 3]) {
    esgFundingServices {
       id
       clientId // client.id
       ...
     }
  }
```

I think the only reason to do that is if we anticipated a need to include some metadata with the report. Otherwise we could definitely just use an array type instead if you prefer.

Reply:
I see. Either is fine. Sounds like an array might be simpler so we don't have to worry about coming up with a cache id for the object.